src/qibolab/dummy/platform.py

```diff
@@ -17,19 +17,21 @@ def create_dummy() -> Platform:
     qubits = {}
     # attach the channels
     for q in range(5):
-        probe, acquisition = f"qubit_{q}/probe", f"qubit_{q}/acquire"
+        probe, acquisition = f"qubit_{q}/probe", f"qubit_{q}/acquisition"
         qubits[q] = Qubit(
             name=q,
-            probe=IqChannel(probe, mixer=None, lo=None, acquisition=acquisition),
-            acquisition=AcquireChannel(acquisition, twpa_pump=pump.name, probe=probe),
-            drive=IqChannel(f"qubit_{q}/drive", mixer=None, lo=None),
-            drive12=IqChannel(f"qubit_{q}/drive12", mixer=None, lo=None),
-            flux=DcChannel(f"qubit_{q}/flux"),
+            probe=IqChannel(name=probe, mixer=None, lo=None, acquisition=acquisition),
+            acquisition=AcquireChannel(
+                name=acquisition, twpa_pump=pump.name, probe=probe
+            ),
```

Comment:
Another point to note is that currently we have the `.acquisition` attribute in the probe channel and the `.probe` attribute in the acquisition channel to link one to the other. If you see for example in https://github.com/qiboteam/qibolab/blob/1940ba26d42f75a62ac083d8bcaa5b28f0b6545d/src/qibolab/components/channels.py#L52
these were temporary solutions until the acquisition business is sorted (done in this PR).

These attributes ended up being quite handy for the QM driver, because we need to map both these channels to the same `element` in the QM config and having the immediate connection through attributes makes it easier. However, the addition of `ChannelId` makes them redundant, since now we can establish the same connection by finding the qubit `q = probe.name.qubit` and then looking for `f"{q}/acquisition"` in the global channel map. This would require updating the QM driver once again, but that's acceptable as we are finalizing the design.

As said before, I am not sure if it is optimal navigating through channels using the qubit, however the previous design was not flawless either (it already had the assumption that a probe should have an acquisition), so it should be fine to switch.